test/inductor/test_aot_inductor.py

```diff
@@ -176,20 +179,51 @@ def forward(self, x, y):
         self.check_model(Repro(), example_inputs)
 
     def test_large(self):
-        class Repro(torch.nn.Module):
-            def __init__(self):
-                super().__init__()
-                self.weight = torch.randn(250112, 512, device="cuda")
-
-            def forward(self, x, y):
-                return x + torch.nn.functional.linear(y, self.weight)
+        model = SimpleLinear((250112, 512))
 
         example_inputs = (
             torch.randn(1, 250112, device="cuda"),
             torch.randn(1, 512, device="cuda"),
         )
         self.check_model(Repro(), example_inputs)
 
+    @unittest.skip("Skip this test, only for local test. SIGABRT is produced.")
+    def test_inf(self):
+        model = SimpleLinear((10, 10))
+
+        x = torch.randn(10, 10, device="cuda")
+        x[0][0] = float("Inf")
+        example_inputs = (
+            x,
+            torch.randn(1, 10, device="cuda"),
+        )
+        expected = model(*example_inputs)
+        actual = AOTInductorModelRunner.run(
+            model,
+            example_inputs,
+            expected,
+            options={"debug_check_inf_and_nan": True},
+        )
+        self.assertTrue(same(actual, expected))
+
+    @unittest.skip("Skip this test, only for local test. SIGABRT is produced.")
+    def test_nan(self):
+        model = SimpleLinear((10, 10))
+
+        x = torch.randn(10, 10, device="cuda")
+        x[0][0] = float("nan")
+        example_inputs = (
+            x,
+            torch.randn(1, 10, device="cuda"),
+        )
+        expected = model(*example_inputs)
+        actual = AOTInductorModelRunner.run(
+            model,
+            example_inputs,
+            expected,
+            options={"debug_check_inf_and_nan": True},
+        )
```

Comment:
Should we assert here?

Reply:
hmmm.. it doesn't really matter.
Line 207 doesn't really run, because the assert in C++ can't be captured by python code. It will just fail out directly. Also, even if we have some mechanism to "continue", I don't think comparison of NaN is meaningful in Python (i.e. `float("nan") != float("nan"`)

Reply:
You could use torch.isnan to do this comparison (call it on each tensor and check the results match) not sure if this is worth it if things fail out before.